PureFitnessV2/Program.cs

```diff
@@ -9,7 +9,9 @@
 
 namespace PureFitnessV2
 {
+#pragma warning disable CA1052 // Static holder types should be Static or NotInheritable
     public class Program
+#pragma warning restore CA1052 // Static holder types should be Static or NotInheritable
     {
         public static void Main(string[] args)
         {
```

Comment:
'public class Program' was throwing a warning of 'Static holder types should be Static or NonInheritable'. Suppressed the issues for now.